modules/mergeday/manifests/install.pp

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+class mergeday::install {
+    include ::config
+    include users::buildduty
+    include packages::mercurial
+    include packages::mozilla::python27
+    include packages::mozilla::py27_virtualenv
+    include packages::mozilla::py27_mercurial
+    include packages::patch
```

Comment:
Hm, unclear to me, why do we need this?

Reply:
It's used as part of the migration scripts: https://hg.mozilla.org/mozilla-central/file/2cca333f546f38860f84940d4c72d7470a3410f4/testing/mozharness/scripts/merge_day/gecko_migration.py#l230

Reply:
Ah, I totally missed that, sorry. Thanks for the pointer!